ledger/testing/randomAccounts.go

```diff
@@ -58,47 +57,62 @@ func RandomNote() []byte {
 	return note[:]
 }
 
-// RandomAccountData generates a random AccountData
+// RandomAccountData generates a random AccountData with no associated resources.
 func RandomAccountData(rewardsBase uint64) basics.AccountData {
 	var data basics.AccountData
 
 	// Avoid overflowing totals
 	data.MicroAlgos.Raw = crypto.RandUint64() % (1 << 32)
+	// 0 is an invalid round, but would be right if never proposed
+	data.LastProposed = basics.Round(crypto.RandUint64() % 10)
+	// 0 is an invalid round, but would be right if never needed a heartbeat
+	data.LastHeartbeat = basics.Round(crypto.RandUint64() % 10)
 
 	switch crypto.RandUint64() % 3 {
 	case 0:
 		data.Status = basics.Online
-		data.VoteLastValid = 10000
+		data.IncentiveEligible = crypto.RandUint64()%5 == 0
 	case 1:
 		data.Status = basics.Offline
-		data.VoteLastValid = 0
-	default:
+	case 2:
 		data.Status = basics.NotParticipating
 	}
 
-	data.VoteFirstValid = 0
+	// Give online accounts voting data, and some of the offline too. They are "suspended".
+	if data.Status == basics.Online || (data.Status == basics.Offline && crypto.RandUint64()%5 == 1) {
```

Comment:
random 20% of offline - just a feels right number?

Reply:
Yeah, it's arbitrary, but I get the impression that we tend to have tests on 20 or so random accounts, so I wanted to pick a number that would likely pop up a few times.